decidim-meetings/app/controllers/concerns/decidim/meetings/component_filterable.rb

```diff
@@ -18,10 +18,10 @@ def default_filter_params
             with_any_date: "upcoming",
             activity: "all",
             with_availability: "",
-            with_any_scope: default_filter_scope_params,
-            with_any_category: default_filter_category_params,
+            with_any_scope: nil,
```

Comment:
check the comment on decidim-budgets/app/controllers/decidim/budgets/projects_controller.rb